test/setup_bundler_test.rb

```diff
@@ -511,6 +514,8 @@ def test_ruby_lsp_rails_is_automatically_included_in_rails_apps
         assert_match('gem "ruby-lsp-rails"', File.read(".ruby-lsp/Gemfile"))
       end
     end
+  ensure
+    Encoding.default_external = original_encoding
```

Comment:
```suggestion
    Encoding.default_external = T.must(original_encoding)
```

Reply:
Can I suggest to extract a helper method for this?
```ruby
def with_default_external_encoding(encoding, &block)
  original_encoding = Encoding.default_external
  begin
    Encoding.default_external = encoding
    block.call
  ensure
    Encoding.default_external = original
  end
end
```
which should need the must and makes the test code read nicer.

Reply:
Yeah, I like that 👍